<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XScreenClass</class>
 <widget class="QWidget" name="XScreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XScreen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xscreen.qrc">
    <normaloff>:/XScreen/logo.png</normaloff>:/XScreen/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#back
{
background-color: rgb(53, 53, 53);
border-radius:15px;
}
#all
{
background-color: rgb(35, 35, 35);
}
#title
{
color:rgb(0, 170, 255);
font:75 15pt &quot;黑体&quot;;
}
#timeLabel
{
color:rgb(0, 170, 255);
font:75 20pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/mini.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="all" native="true">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>50</y>
      <width>891</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
	background-image: url(:/XScreen/record_normal.png);
}
QPushButton:hover
{
	background-image: url(:/XScreen/record_hot.png);
}
QPushButton:pressed
{
	background-image: url(:/XScreen/record_pressed.png);
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>D:/video</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>1280</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>720</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>000:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>维护：ysrs</string>
     </property>
    </widget>
    <widget class="QLabel" name="widthLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>25</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>宽：</string>
     </property>
    </widget>
    <widget class="QLabel" name="heightLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>25</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>高：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fpsEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="fpsLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>25</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>fps：</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>XScreen屏幕录像机 v1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/logo.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xscreen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>Record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Record()</slot>
 </slots>
</ui>
